llvm/tools/vfabi-demangle-fuzzer/vfabi-demangler-fuzzer.cpp

```diff
@@ -27,15 +27,17 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
   // present. We need to make sure we can even invoke
   // `getOrInsertFunction` because such method asserts on strings with
   // zeroes.
```

Comment:
I'm not sure about this test. The comment is definitely wrong now but does the test still make sense?

Reply:
Resolved.